src/main/java/org/alliancegenome/curation_api/dao/ontology/GoTermDAO.java

```diff
@@ -12,4 +16,19 @@ protected GoTermDAO() {
 		super(GOTerm.class);
 	}
 
+	public Map<String, Long> getAllGOIds() {
```

Comment:
Here is already an method that does this.

Reply:
This is a method that only returns a lean object, basically a map that associates the GO curie with the GOTermID.